db/models/review.js

```diff
@@ -1,19 +1,23 @@
+'use strict'
+const Sequelize = require('sequelize')
+const db = require('./index.js')
+
 module.exports = db => db.define('reviews', {
-  title: STRING,
-  content: TEXT,
+  title: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  content: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
   num_stars: {
-    type: FLOAT,
+    type: Sequelize.DECIMAL(1, 2),
```

Comment:
Hi Jeff, yes the test was what confused me when I was writing this file. 
Sure the review should take only 1-5stars, I'll modify the num_star type in this model and the review test too.